content/TrueCommand/Clusters.md

```diff
@@ -85,38 +85,45 @@ There is a single option on this page.
 ### Configure Cluster
 
 Successfully creating a cluster adds a cluster widget to the **Clusters** screen and opens options to configure the new cluster.
-These options split into three screens: **SMB Network**, **Active Directory**, and **Confirmation**.
+These options split into four screens: **VIPs**, **Accociate VIPs**, **Active Directory**, and **Confirmation**.
 
-#### SMB Network
+#### VIPs
 
-The SMB Network page has options and fields added for each SCALE system in the cluster.
+The VIPs page has options and fields added for each SCALE system in the cluster.
 
-![ConfigureClusterSMBNetwork](/images/TrueCommand/2.2/ConfigureClusterSMBNetwork.png "Configure Cluster SMB Network")
-
-{{< expand "Option descriptions (click to expand)" "v" >}}
+![ConfigureClusterSMBNetwork](/images/TrueCommand/2.3.1/ConfigureClusterSMBNetwork.png "Configure Cluster SMB Network")
 
 | Setting | Description |
 |---------|-------------|
 | **NEXT** | Click button to continue to the next page. |
-| **Skip this step** | Select checkbox to finalize the cluster configuration without registering a network to use for SMB sharing. Selecting this permanently removes the ability to share data stored in this cluster. |
-| **Interface** | Dropdown list shows SCALE system network interface to use for SMB traffic. For best functionality, choose an interface that can communicate between the individual SCALE systems, TrueCommand environment, Active Directory, and client systems that need to access data shared from this cluster. |
-| **Address** | Dropdown list shows IP address used for clients to access the SCALE system. |
-| **Netmask** | Dropdown list shows netmask for the IP address. |
+| **ADD** | Adds another line for IPs and netmasks. |
```

Comment:
Where does it add this line? Below the xyz on the screen? Tell them where so they know what to expect.